cfme/tests/cloud/test_provisioning.py

```diff
@@ -207,6 +207,107 @@ def _finished():
         num_sec=600, delay=5, message="VM {} becomes visible".format(vm_name))
 
 
+@pytest.mark.uncollectif(lambda provider: provider.type != 'ec2' or current_version() < "5.7")
```

Comment:
REQUIRED: provider.type is deprecated please use one_of method instead.
"not provider.one_of(EC2Provider)"